packages/embed/package.json

```diff
@@ -5,9 +5,18 @@
   "main": "build/index.js",
   "author": "Typeform",
   "license": "LGPL-3.0-only",
+  "engines": {
+    "node": ">=14"
+  },
   "scripts": {
     "build": "tsc",
     "dev": "tsc -w",
-    "lint": "eslint src --ext .js,.ts,.jsx,.tsx --max-warnings=0"
+    "lint": "eslint src --ext .js,.ts,.jsx,.tsx --max-warnings=0",
+    "test": "jest"
+  },
+  "devDependencies": {
+    "@types/jest": "^26.0.20",
```

Comment:
Should we put dependencies here or to the root? How to do this? I am not familiar with lerna. @prilutskiy please advise.

Reply:
Let's keep everything in the root, so the same packages' versions could be reused across all projects